prow/gerrit/client/client.go

```diff
@@ -228,7 +229,7 @@ func (c *Client) GetBranchRevision(instance, project, branch string) (string, er
 		return "", fmt.Errorf("not activated gerrit instance: %s", instance)
 	}
 
-	res, _, err := h.projectService.GetBranch(project, branch)
```

Comment:
Is project also user provided? If so then shouldn't that be too?

Reply:
see https://github.com/andygrunwald/go-gerrit/blob/master/projects_branch.go#L75, the project is properly escaped in upstream, but not branch :-(

Reply:
Fix upstream?

Reply:
yeah I have a fix opened (mentioned in PR body)